### PR TITLE
add copy button to sprite_frames_editor

### PR DESCRIPTION
**Pull request by sersoong**
Add a copy button to sprite_frames_editor.
Easy to duplicate sprite in sprite frames editor,or copy to anywhere..

### CODE REVIEW

**Review by 27thLiz — APPROVED** [review]

File: editor/plugins/sprite_frames_editor_plugin.cpp [hunk 1]
```diff
@@ -199,6 +199,21 @@ void SpriteFramesEditor::_paste_pressed() {
 	undo_redo->commit_action();
 }
 
+void SpriteFramesEditor::_copy_pressed(){
+
+	ERR_FAIL_COND(!frames->has_animation(edited_anim));
+
+	if(tree->get_current() < 0)
```
Comment (akien-mga) [inline editor/plugins/sprite_frames_editor_plugin.cpp]: Not sure why `clang-format` did not complain, but this doesn't follow our style guide, it should be:
```
if (tree->get_current() < 0)
	return;
Ref<Texture> r = frames->get_frame(edited_anim, tree->get_current());
if (!r.is_valid()) {
	return;
}
```

Can you amend the commit with `git commit --amend` and update the PR with `git push --force`? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

**Comment by akien-mga** [comment]
Note that this question mark next to your commit's author name and the lack of image means that the email address you used for git locally is not linked to your GitHub account. You might want to fix it by adding that address as secondary GitHub address, thus claiming back your commits made under this identity.
image

**Comment by akien-mga** [comment]
Could you also make the equivalent PR for the master branch if relevant?

File: editor/plugins/sprite_frames_editor_plugin.cpp (hunk 1 above)
Comment (akien-mga) [reply editor/plugins/sprite_frames_editor_plugin.cpp]: See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html regarding the code style and how to use `clang-format` locally.

**Comment by sersoong** [comment]
I will make it a little later..

File: editor/plugins/sprite_frames_editor_plugin.cpp (hunk 1 above)
Comment (sersoong) [reply editor/plugins/sprite_frames_editor_plugin.cpp]: clang-format config error...
capture

Comment (akien-mga) [reply editor/plugins/sprite_frames_editor_plugin.cpp]: What's your `clang-format` version? We use 3.9.1 currently.

**Pull request merged**